readme.txt

```diff
@@ -1,23 +1,26 @@
 === Cookie Wow WordPress ===
 Contributors: Cookie Wow
-Tags: cookiewow
-Requires at least:
+Tags: cookie, PDPA
+Requires at least: 4.9.16
```

Comment:
The screenshot below shows the plugin has been tested with WordPress 4.9.16.

<img width="800" src="https://user-images.githubusercontent.com/4145121/109455629-a149ab00-7a89-11eb-8646-f720f615096a.png">